parcels/field.py

```diff
@@ -404,7 +404,8 @@ def show(self, with_particles=False, animation=False, show_time=0, vmin=None, vm
             import matplotlib.animation as animation_plt
             from matplotlib import rc
         except:
-            raise RuntimeError("Visualisation not possible: matplotlib not found!")
+            print("Visualisation is not possible. Matplotlib not found.")
```

Comment:
I'm not sure dropping the `RuntimeError` here is the right thing to do. As far as I understand it this only breaks testing when invoked from the plotting script or the notebooks. So can we not use an explicit `try/except` to simply make these more robust, but leave the low-level errors in place?